tests/resources/serializers/test_bibtex_serializer.py

```diff
@@ -178,23 +178,25 @@ def test_serialize_publication_conferencepaper(running_app, updated_minimal_reco
 
     assert serialized_record == expected_data
 
-    # Force serialization into 'proceedings'
+    # Force serialization into 'misc'
     del updated_minimal_record["custom_fields"]["imprint:imprint"]
     serialized_record = serializer.serialize_object(updated_minimal_record)
 
     expected_data = "\n".join(
         [
-            "@proceedings{brown_2023_abcde-fghij,",
+            "@misc{brown_2023_abcde-fghij,",
             "  author       = {Name and",
             "                  Troy Inc.},",
```

Comment:
Note the this part of the test did NOT have an assertion, and that the expected data was not correct (proceedings do NOT have an author).